documentation/book/cluster-operator.adoc

```diff
@@ -515,6 +518,27 @@ Furthermore, containers doing lots of disk I/O (such as Kafka broker containers)
 for use as operating system page cache. On such containers, the request memory should be substantially more than the
 memory used by the JVM.
 
+[[kafka_rack]]
+===== Kafka rack
+
+It is possible to enable Kafka rack-awareness by specifying a JSON string for the `kafka-rack` field in the cluster ConfigMap.
```

Comment:
I think Rack awareness is something not understandable to everyone. We should either explain a bit more or link to Kafka docu?

Reply:
I don't think that we have to put more documentation about how Kafka works in the Strimzi project. I'll put a link to the official Kafka documentation if someone wants to get more information about rack  awareness.